InnoSetup/CnCNet5_YR_Installer.iss

```diff
@@ -55,10 +55,37 @@ Name: "english"; MessagesFile: "compiler:Default.isl"
 
 [InstallDelete]
 ;Type: files; Name: "{app}\wsock32.dll"; MinVersion: 6.2
-Type: files; Name: "{app}\version"
+;Type: files; Name: "{app}\version"
+Type: files; Name: "{app}\Resources\Allied Theme\CnCNetLobby.ini"
+Type: files; Name: "{app}\Resources\Allied Theme\GenericWindow.ini"
+Type: files; Name: "{app}\Resources\Allied Theme\LANLobby.ini"
+Type: files; Name: "{app}\Resources\Allied Theme\LoadingScreen.ini"
+Type: files; Name: "{app}\Resources\Allied Theme\MultiplayerGameLobby.ini"
+Type: files; Name: "{app}\Resources\Allied Theme\OptionsWindow.ini"
+Type: files; Name: "{app}\Resources\Allied Theme\SkirmishLobby.ini"
+Type: files; Name: "{app}\Resources\Soviet Theme\CnCNetLobby.ini"
+Type: files; Name: "{app}\Resources\Soviet Theme\LANLobby.ini"
+Type: files; Name: "{app}\Resources\Soviet Theme\LoadingScreen.ini"
+Type: files; Name: "{app}\Resources\Soviet Theme\MultiplayerGameLobby.ini"
+Type: files; Name: "{app}\Resources\Soviet Theme\OptionsWindow.ini"
+Type: files; Name: "{app}\Resources\Soviet Theme\SkirmishLobby.ini"
+Type: files; Name: "{app}\Crate.sno"
+Type: files; Name: "{app}\crate.sno"
+Type: files; Name: "{app}\crate.des"
+Type: files; Name: "{app}\crate.lun"
+Type: files; Name: "{app}\crate.tem"
+Type: files; Name: "{app}\crate.ubn"
+Type: files; Name: "{app}\crate.urb"
+Type: files; Name: "{app}\wcrate.des"
+Type: files; Name: "{app}\wcrate.lun"
+Type: files; Name: "{app}\wcrate.sno"
+Type: files; Name: "{app}\wcrate.tem"
+Type: files; Name: "{app}\wcrate.ubn"
+Type: files; Name: "{app}\wcrate.urb"
+Type: files; Name: "{app}\expandspawn09.mix"
```

Comment:
These are all the files from the predelete of the updater, since we don't know what version the person might be on when they use the installer.